include/z64camera.h

```diff
@@ -3,10 +3,11 @@
 
 #include "ultra64.h"
 #include "z64cutscene.h"
+#include "z64math.h"
 #include "z64save.h"
 
 // these two angle conversion macros are slightly inaccurate
-#define CAM_DEG_TO_BINANG(degrees) (s16)((degrees) * 182.04167f + .5f)
+#define CAM_DEG_TO_BINANG(degrees) (s16)TRUNCF_BINANG((degrees) * 182.04167f + .5f)
```

Comment:
You can drop the `s16` cast since it is included on the macro itself

Reply:
Keeping it was intentional, to make it clear this evaluates to a s16, which is typically important in binang comparisons

I can remove them though if `TRUNCF_BINANG` is enough to indicate that